torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -3035,8 +3035,8 @@ std::tuple<at::Tensor, at::Tensor> XLANativeFunctions::var_mean(
                          bridge::AtenFromXlaTensor(std::get<1>(results)));
 }
 
-at::Tensor XLANativeFunctions::view(const at::Tensor& self,
-                                    at::SymIntArrayRef sym_size) {
+at::Tensor XLANativeFunctions::view_symint(const at::Tensor& self,
```

Comment:
To catch up: 

+ I understand this PR only enables `SymInt` for `view`
+ I don't expect `view` is ready for Dynamic Shape through this change - i.e. more work would be needed for DS enablement. The same applies to `empty`, `narrow_copy`.

@Krovatkin @JackCaoG please confirm (or correct me if inaccurate).

Reply:
Yeah, all this does is change the calling convention; but the insides still assert that the int is not symbolic. Once the IR side is ready, you can just modify this function to track it.

Future SymInt'ified schemas will not get updates like this, because we subsequently added an interlock (the `symint:` field in the yaml) which controls which calling convention you get. But there's a few stragglers like this from before we had this interlock.

Reply:
> Once the IR side is ready, you can just modify this function to track it.

Is this part of an existing PR already? If so, please point me to it so I follow your changes accordingly.

@ezyang

Reply:
no, that was up to you guys :)